doc/rados/configuration/bluestore-config-ref.rst

```diff
@@ -134,54 +149,57 @@ Finally, create the 4 OSDs with ``ceph-volume``:
    ceph-volume lvm create --bluestore --data ceph-block-2/block-2 --block.db ceph-db-0/db-2
    ceph-volume lvm create --bluestore --data ceph-block-3/block-3 --block.db ceph-db-0/db-3
 
-These operations should end up creating four OSDs, with ``block`` on the slower
-rotational drives with a 50 GB logical volume (DB) for each on the solid state
-drive.
+After this procedure is finished, there should be four OSDs, ''block'' should
```

Comment:
Should those single quotes be backticks?

Reply:
They sure should.

Accepted.